OM-481 | Implement common accessibility considerations

Jokinen opened the pull request:
* I label navigation menus more distinctly. Here I was not sure if it's good UX to have separate navs for language and user. Maybe not?
* I synced html lang attribute with selected language
* I added a skip to content link

Jokinen committed:
Remove unclear section

The purpose of this section is not entirely clear, but I assume that it
stands for a navigation region. If so, a more proper way to label this
part of the UI would be a nav tag or, or role="navigation".

Jokinen committed:
Label navigation menus

Jokinen committed:
Sync html lang attribute with selected language

Jokinen committed:
Add skip to content link

Jokinen commented:
Hmn, I don't know why the deployment fails.

I have an old version of tokens on this change. I am going to upgrade.

Jokinen committed:
Update hds-design-tokens

tuovinensanttu reviewed (approved):
Code wise everything is looking good 👍

badrange commented on src/common/accessibilityShortcuts/AccessibilityShortcuts.tsx:
```diff
@@ -0,0 +1,27 @@
+        className={classNames(styles.srOnlyFocusable, styles.accessibilityLink)}
+        href={mainContentHref}
+      >
+        {t('skipToContent')}
```
At some point it might be a good idea to go through the translation keys

Jokinen committed:
Move react-helmet types to dependencies

These types would be ignored for static builds when added through
devDependencies.
